packages/preact/hooks/src/useFacet.ts

```diff
@@ -70,6 +70,13 @@ export function useFacet(facet: SearchTermsFacet, options?: UseFacetOptions) {
   const [active, setActive] = useState(initialActive)
   const { toggleProductFilter } = useActions()
 
+  // Reset active state when all filters are cleared and no explicit active option is set
+  useEffect(() => {
+    if (selectedFiltersCount === 0 && !options?.active) {
+      setActive(false)
```

Comment:
this reset should happen when all filters are cleared, not when the filters associated to this fact are cleared
shared state with useProductFilters might be needed to solve this